resources/flaky-github-comment-markdown.template

```diff
@@ -2,7 +2,8 @@
 ## :bug: Flaky test report
 :snowflake: The following tests failed but also have a history of flakiness and may not be related to this change: 
     <% flakyTests?.each{ k,v -> %>
-* **Name**: `${k}` ${ (v?.trim()) ? "reported in the issue #" + v : 'has not been reported yet.'}<%}%>
+<% url = (v?.startsWith('https:')) ? "${v}" : "#${v}"%>
+* **Name**: `${k}` ${ (v?.trim()) ? "reported in the issue " + url : 'has not been reported yet.'}<%}%>
```

Comment:
`githubCreateIssue` and `lookForGitHubIssues` steps return different outputs. Let's support those cases